tripleo-heat-templates-diff/apply_patch.py

```diff
@@ -40,44 +41,69 @@
 PRE_VERSION_3_DIFF = "diff_OSPD10_5.3.3-1"
 POST_VERSION_3_DIFF = "diff_OSPD10_5.3.10-17"
```

Comment:
wouldn't it be nice of this info is put in some config file rather than in this source file

Reply:
Can be done when I combine the document. I will put them in mind